client/boot/index.js

```diff
@@ -165,14 +166,7 @@ function boot() {
 function renderLayout( reduxStore ) {
 	const Layout = require( 'controller' ).ReduxWrappedLayout;
 
-	const layoutElement = React.createElement( Layout, {
-		store: reduxStore
-	} );
-
-	ReactDom.render(
-		layoutElement,
-		document.getElementById( 'wpcom' )
-	);
+	renderWithReduxStore( React.createElement( Layout, { store: reduxStore } ), 'wpcom', reduxStore );
```

Comment:
Yeah, it's quite obvious from [these lines](https://github.com/Automattic/wp-calypso/blob/c88d20a077df3b2ee0676a39e211a1ec57f9d3b4/client/boot/index.js#L167-L169) that we're wrapping `Layout` twice with `ReduxProvider`.

> Appears we should have updated the `controller/index.node.js` file to stop wrapping with `ReduxProvider` with these changes?

No, that's an SSR'd single-tree, so we need it at the top of the component hierarchy.